packages/moonstone/styles/mixins.less

```diff
@@ -270,9 +270,9 @@
 }
 
 .moon-sub-header-text() {
-	.moon-text-base(@moon-sub-header-font-size);
+	.moon-text-base(@moon-header-title-below-font-size);
 	font-weight: @moon-sub-header-font-weight;
-	font-size: @moon-sub-header-font-size;
+	font-size: @moon-header-title-below-font-size;
```

Comment:
Should we change the rest of `sub-header` to `header-title-below`?

Reply:
This would be wise to just unilaterally fix everywhere. It's labor-intensive but does raise the quality and consistency bar. I'll see what I can do in a reasonable amount of time.

Reply:
I could follow up with this work with the Dialog ticket too if you need it.

Reply:
This sort of cleanup would be great to get done now, before 3.0.0 finalizes. If you do remove any variables, or even rename any, be sure to include them in the 3.0 migration guide (referenced above).

I know you already know, but for historical record-keeping purposes, the current LESS variable naming convention is as follows:
```
@{theme}-{component}-[{state(s)}-]{cssProperty|abstractedName}
```
* The only allowed (known) safe abbreviation is for "background"->"bg". Otherwise, everything else should be spelled out.
* `color` rule can be conditionally referenced as `-text-color` if the commonly accepted content is indeed text. It may be shortened to simply its rule name, in cases like `Icon` where technically the thing you're adjusting is the text color, but you're actually modifying the color of a glyph and including "text" isn't particularly intuitive.
* States, if present fall between the component name and the CSS rule name.
   * If there are multiple, favoring design-time states first and runtime states second, with alphabetical being a tie-breaker. Ex:
```
@moon-button-bg-color;
@moon-button-focus-bg-color;
@moon-button-selected-bg-color;
@moon-button-selected-focus-bg-color;
@moon-button-disabled-bg-color;
@moon-button-disabled-focus-bg-color;
@moon-button-disabled-selected-bg-color;
@moon-button-disabled-selected-focus-bg-color;
```